Sources/Sentry/SentrySession.m

```diff
@@ -26,7 +26,7 @@ - (instancetype)init {
 }
 
 - (instancetype)initWithJSONObject:(NSDictionary *)jsonObject {
-    if (self = [super init]) {
+    if (self = [self init]) {
```

Comment:
@HazAT I just realized this. Probably following the warning since the other init is `NS_DESIGNATED_INITIALIZER` but in this case we don't want to run it since it initializes defaults (like timestamp) and we want to restore the exact values from the JSON. I'll change it back and add a comment